Frameworks/UIKit/XamlControls.h

```diff
@@ -121,6 +121,12 @@ WXCGrid* CreateLabel();
 WXCTextBlock* GetLabelTextBlock(WXCGrid* labelGrid);
 
 ////////////////////////////////////////////////////////////////////////////////////
+// ScrollViewer
+////////////////////////////////////////////////////////////////////////////////////
+WXCGrid* CreateScrollViewer();
+WXCScrollViewer* GetScrollViewer(WXCGrid* labelGrid);
```

Comment:
yeah. The creation part is a little confusing - meaning it returns Grid instead of Scrollview.  But i guess that is the price that we need to pay before projection is ready.

---
In reply to: [106713161](https://github.com/Microsoft/WinObjC/pull/2257#discussion_r106713161) [](ancestors = 106713161)